typescript/03 barrels/src/app.tsx

```diff
@@ -27,14 +25,15 @@ export class App extends React.Component<Props, State> {
   public render() {
     return (
       <>
-        <MuiThemeProvider>
```

Comment:
Dont remove `MuiThemeProvider`

Reply:
I think on @next is neeeded to be removed if you are not setting your custom them

Reply:
Ahh ok, so we need to update Readme

Reply:
Nvm, it's updated. With all `+ -` it's difficult to review it